documentation/source/user_section/tutorials.rst

```diff
@@ -69,7 +69,7 @@ The past courses handouts are listed below:
 -  `SCT course (v3.0_beta1), Montreal, 2016-04-19`_
 -  `SCT Hands-on Workshop (v2.0.4), Toronto, 2015-06-15`_
 
-.. _SCT course (v4.2.1), London, 2020-01-21: https://www.icloud.com/keynote/0th8lcatyVPkM_W14zpjynr5g#SCT%5FCourse%5F20200121
+.. _SCT course (v4.2.1), London, 2020-01-21: https://drive.google.com/file/d/1rn6srrQRLmO6cMHJ9OSrOBD7xjMm6uzn/view?usp=sharing
```

Comment:
Note: This link includes slides only. If we want to include presenter notes, someone with Keynote on their Mac will need to export the slides with presenter notes. (I couldn't figure out how to do this via iCloud only.)

Reply:
I'm happy to do that once the little issues in the slides are fixed

Reply:
Ah! So, this is the link to the 2020 slides (i.e. last year's). So, this one is fine to export now -- my plan was to add the 2021 slides will be added in a follow-up PR.

Reply:
here you go:
https://drive.google.com/file/d/1TZireJ6yhV8q7PbKKXyXg7Heov9-YJMu/view?usp=sharing

Reply:
(also added in the same gdrive folder)